8.cytopick_analysis/notebooks/find_prediction_images.ipynb

```diff
@@ -1,1914 +1,1911 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    for key in tqdm(dict_of_subset_dfs):
</pre><p>Provide a general comment of what the for loop is doing. This will give the reader an abstract idea.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>